dataformat-json-jackson/src/test/java/org/camunda/spin/javascript/json/tree/JsonTreeMapObjectToJsonJavascriptTest.java

```diff
@@ -16,18 +16,18 @@
  */
 package org.camunda.spin.javascript.json.tree;
 
-import org.camunda.spin.impl.test.ScriptEngine;
 import org.camunda.spin.json.tree.JsonTreeMapObjectToJsonScriptTest;
+import org.camunda.spin.test.ScriptEngine;
 import org.junit.Ignore;
 
 /**
- * Note: Jackson is apparently not able to map Nashorn's internal representations
+ * Note: Jackson is apparently not able to map graal.js's internal representations
```

Comment:
The comment mentions `graal.js` but the `@ScriptEngine` annotation receives `nashorn` as an input argument. Is this correct? It's a bit confusing.

Reply:
It's supposed to run on Graal.js